internal/commands/resolve.go

```diff
@@ -40,6 +42,28 @@ func ResolveIncidentDialog(client bot.Client, triggerID string) error {
 		Subtype: slack.InputSubtypeURL,
 	}
 
+	postMortemMeeting := &slack.DialogInputSelect{
+		DialogInput: slack.DialogInput{
+			Label:       "Can i schedule a meeting of Post Mortem?",
+			Name:        "post_mortem_meeting",
+			Type:        "select",
+			Placeholder: "Post Mortem Meeting",
+			Optional:    false,
```

Comment:
Why we don't set this as an optional field, and the default is false?

Reply:
As it is optional, it allows the user not to fill in a value. When the user sends nothing, the default value is not taken into account by the Dialog component.

![image](https://user-images.githubusercontent.com/36651634/84937809-b379e980-b0b2-11ea-8b46-e9755c5f09e7.png)

The default value is used when filling in the field. So I left it as mandatory.
![image](https://user-images.githubusercontent.com/36651634/84938140-27b48d00-b0b3-11ea-858f-8cadf81b9a31.png)

Reply:
Agreed